transport.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'constants'
+
+class Transport
+  attr_accessor :weight, :speed, :available, :distance
+
+  def initialize(weight, speed, available, distance = nil)
+    @weight = weight
+    @speed = speed
+    @available = available
+    @distance = distance
+  end
+
+  def delivery_time
```

Comment:
Technical task specifies that `delivery_time` gets distance as its argument, please, fix.

Reply:
Done.